ipynb/android-wloads/Android_PCMark_Tests.ipynb

```diff
@@ -0,0 +1,440 @@
+      "2016-03-23 13:33:08,017 INFO    :         Target - Connecting Android target [{'device': '00b1346f0878ccb1'}]\n",
+      "2016-03-23 13:33:08,633 INFO    :         Target - Initializing target workdir:\n",
+      "2016-03-23 13:33:08,634 INFO    :         Target -    /data/local/tmp/devlib-target\n",
+      "2016-03-23 13:33:10,750 INFO    :         Target - Topology:\n",
+      "2016-03-23 13:33:10,751 INFO    :         Target -    [[0]]\n",
+      "2016-03-23 13:33:10,777 WARNING :        TestEnv - Wipe previous contents of the results folder:\n",
+      "2016-03-23 13:33:10,778 WARNING :        TestEnv -    /home/micgio01/work/lisa/results/AndroidN5X_PCMark\n",
+      "2016-03-23 13:33:10,779 INFO    :        TestEnv - Set results folder to:\n",
+      "2016-03-23 13:33:10,780 INFO    :        TestEnv -    /home/micgio01/work/lisa/results/AndroidN5X_PCMark\n",
+      "2016-03-23 13:33:10,781 INFO    :        TestEnv - Experiment results available also in:\n",
+      "2016-03-23 13:33:10,782 INFO    :        TestEnv -    /home/micgio01/work/lisa/results_latest\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Initialize a test environment using:\n",
+    "# the provided target configuration (my_target_conf)\n",
+    "# the provided test configuration   (my_test_conf)\n",
+    "te = TestEnv(target_conf=my_target_conf, test_conf=my_tests_conf)\n",
+    "target = te.target"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Support Functions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This set of support functions will help us running the benchmark using different CPUFreq governors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def set_performance():\n",
+    "    target.cpufreq.set_all_governors('performance')\n",
+    "\n",
+    "def set_powersave():\n",
+    "    target.cpufreq.set_all_governors('powersave')\n",
+    "\n",
+    "def set_interactive():\n",
+    "    target.cpufreq.set_all_governors('interactive')\n",
+    "\n",
+    "def set_sched():\n",
+    "    target.cpufreq.set_all_governors('sched')\n",
+    "\n",
+    "def set_ondemand():\n",
+    "    target.cpufreq.set_all_governors('ondemand')\n",
+    "    \n",
+    "    for cpu in target.list_online_cpus():\n",
+    "        tunables = target.cpufreq.get_governor_tunables(cpu)\n",
+    "        target.cpufreq.set_governor_tunables(\n",
+    "            cpu,\n",
+    "            'ondemand',\n",
+    "            **{'sampling_rate' : tunables['sampling_rate_min']}\n",
+    "        )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# CPUFreq configurations to test\n",
+    "confs = {\n",
+    "#    'performance' : {\n",
```

Comment:
Again, let's try to have a notebook which always runs by default.
Thus I would suggest to keep not commented only two of the governors which are more likely to be available on a generic target. What about: performance and interactive.
This should give also an interesting analysis on possible/maximum speedup ;-)